port/port_win.h

```diff
@@ -32,9 +32,16 @@
 #define STORAGE_LEVELDB_PORT_PORT_WIN_H_
 
 #ifdef _MSC_VER
+#if !(_MSC_VER >= 1900)
```

Comment:
I think that would be clearer as `_MSC_VER < 1900`.

Reply:
It looks like https://github.com/bitcoin-core/leveldb/pull/7 has extended fix for it.